snap/snapcraft.yaml

```diff
@@ -0,0 +1,70 @@
+name: zed-editor
+title: Zed
+base: core24
+version: '0.149.5'
+summary: Code at the speed of thought
```

Comment:
This can be imported from the appstream meta-data so to avoid having to maintain a special copy. Having said that I have not done this in a long while so I don't recall the syntax from the top of my head.

Reply:
Same as above. We can adopt a lot more fields from appstream metadata as I implemented those in snapcraft. But that's will just complicated things even more.